style/base.css

```diff
@@ -0,0 +1,5 @@
+/*
```

Comment:
Do we need this empty css?

Reply:
Good question. I left it because it's in the original extension template, however I can check if we can get rid of it.

Reply:
We can get rid of it :)